### PR TITLE
fix(issue-326): add missing button to dashboard detail page

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

**Review by minseolee — APPROVED** [review]
It looks good for me!
I think @piggggggggy 's review would be very helpful.

**Review by piggggggggy — APPROVED** [review]
LGTM!!

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue [hunk 1]
```diff
@@ -178,7 +180,10 @@ const handleClickGoBackButton = () => {
         deleteModalState.visible = true;
         return;
     }
-    SpaceRouter.router.go(-1);
+    SpaceRouter.router.replace({
+        name: dashboardDetailOriginState.isProjectDashboard ? DASHBOARDS_ROUTE.PROJECT.CUSTOMIZE._NAME : DASHBOARDS_ROUTE.WORKSPACE.CUSTOMIZE._NAME,
+        params: { dashboardId: dashboardDetailState.dashboardId ?? '' },
+    });
```
Comment (piggggggggy) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue:186]: 🫶🏻

**Review by piggggggggy — APPROVED** [review]
Gooooood~!!

**Pull request merged**